src/components/Home.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { loading } from '../features/country/countrySlice';
+import IconArrowRightCircle from './IconArrowRightCircle';
+
+const Home = () => {
+  const data = useSelector((state) => state.country);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(loading(false));
+  });
+
+  return (
+    <div className="home">
+      <ul className="countries">
+        {
+                data.country.map((item, index) => (
+                  <li key={item.name.common}>
+                    <Link
+                      className={((index + 2) % 2) === 0 ? 'country' : 'country  pair-red'}
+                      to="/details"
+                      state={{ country: item.name.common }}
+                    >
+                      <div className="flag-items">
+                        <img className="flag" src={item.flags.png !== undefined ? item.flags.png : 'Flag not avaible'} alt={item.name.official} />
+                        <IconArrowRightCircle />
+                      </div>
+
+                      <div className="linke-div">
+                        <p className="crt">{item.name.common.toUpperCase()}</p>
+                        <p className="crt-sub">
+                          AREA :
+                          {` ${item.area}`}
+                        </p>
+                        <p className="crt-sub">
+                          POPULATION :
+                          {` ${item.population}`}
+                        </p>
+                      </div>
+                    </Link>
+                  </li>
+                ))
+            }
+      </ul>
+    </div>
+  );
+};
```

Comment:
- [ ] Your design when the app is cloned is different from when viewed from the live demo link. Kindly note that both apps should not have any disparities as the deployed version should be a copy of the local app.

- [ ] The local app design does not follow the requirement. You are required to use one main color as your theme, but you have two colors. Kindly choose one main color and stick to it, per the requirement.

- [ ] For whichever color you choose, kindly ensure you implement the checkerboard pattern, as in the deployed app.

  Deployed app design
  ![image](https://github.com/kerlos-bahaa/React-capstone/assets/81468675/ba2babe2-8993-47bf-a508-9dcd74cc1eec)

  Local app design
  ![image](https://github.com/kerlos-bahaa/React-capstone/assets/81468675/d40634bc-4e4a-43d5-a154-fd5cad9d2066)